modules/ROOT/pages/parse-template-reference.adoc

```diff
@@ -70,16 +70,69 @@ include::{examplesdir}/responseHtml.template[]
 include::{examplesdir}/parse-template-reference.xml[]
 ----
 
-To use the application in this example, you must send an HTTP request with a URL that includes an ID query parameter, such as `http://localhost:8081/getEmployee?id=12345`.
+To use the application in this example, you must send an HTTP request with a URL that includes an ID query parameter, such as `+http://localhost:8081/getEmployee?id=12345+`.
 
-==== Special Characters
+== Special Characters
 
-Parse Template supports the use of nested expression and literals inside expressions. There are some characters for special uses:
+Parse Template supports the use of expressions within a template and the use of literals within these expressions. The template recognizes certain special characters, such as the sequence `\#[` and quotations marks. You can escape such characters to treat them as regular characters.
 
-* An expression, which begins with the sequence `+#[+`, contains nested expressions. Inside an expression, the character `[` defines the start of an expression. The character  `[` does not need a preceding `+#+` . The character `[` is treated as a special character inside an expression.
-* Inside an expression, single quotes are used both to quote text and to treat text as a literal. So, the expression `+#['[']+` is a valid expression that evaluates to `[`. Outside expressions, single quotes are used as literals.
-* To avoid using a single quote to quote text inside an expression, add a backslash to escape it. For example `+#['this isn\'t an expression']+`.
-* Outside an expression, it is not necessary to quote characters `#` and `[` unless they are together. For example to create an expression just write `+\#[ the-expression-to-resolve ]+`.
+* `#[an-expression]`:
++
+Within a template, use the characters `+#[]+` to designate an expression. For example, a template can contain expressions within HTML elements, such as `<td>#[upper("mulesoft")]</td>`, which resolves to `<td>MULESOFT</td>`. Empty expressions (`+#[]+`) return an error, but empty strings, such as `+#['']+` or `+#[""]+` are valid expressions.
+
+* Sub-expressions:
++
+To resolve a sub-expression that is surrounded by quotation marks, you must surround the sub-expression with another `+#[]+`. For example, the three elements in the template snippet below contain the same sub-expression (`upper("world")`), either inside or outside of quotation marks. The top two return the same results. The bottom two do not process `upper("world")` because the sub-expression is inside the quotation marks but not surrounded by another `+#[]+`.
```

Comment:
The top two / The bottom two > The **first** two / The **last** two. 
I don't think this is wrong, but I feel it's better to mention the items by order rather than position, please check and decide if this change is necessary.

Reply:
done